<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectedUserDataDialog</class>
 <widget class="QDialog" name="selectedUserDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>724</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Selected user data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Selected user data</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update user data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="todayDateSelectedPage">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Return document page</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Latest update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="latestUpdateLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>User ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userIdLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PasswordLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Email address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>First name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Last name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="genderLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mobile number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mobileLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Vaccine status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="vaccineLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Date of birth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="birthdayLineEdit">
          <property name="locale">
           <locale language="English" country="NewZealand"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addressLineEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="3">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Note</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="noteLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Covid-19 test result</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="testResultPushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update result</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>808</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="testResultTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Vaccine record</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="VaccineRecordPushButton">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update record</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>868</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="vaccineRecordTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>selectedUserDataDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1145</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>1275</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
